Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -45,7 +49,14 @@ public class BTShopperInsightsClient {
                 "/v2/payments/find-eligible-methods",
                 parameters: postParameters,
                 httpType: .payPalAPI
-            ) { json, response, error in
+            ) { json, _, error in
+                Task {
+                    if let error {
+                        self.notifyFailure(with: error)
+                        throw error
+                    }
+                }
+                self.notifySuccess(with: BTShopperInsightsResult())
```

Comment:
I know this is just a default case until we start messing with the API default responses, but in this case we want to make sure we fire an analytic + give the merchant back a result

```
let result = BTShopperInsightsResult()
self.notifySuccess(with: BTShopperInsightsResult())
return result
```

Reply:
This _might_ run into some async-await error though, we'll have to mess with the Task maybe.

Reply:
ahhh gotcha, addressed in a29756160275030f202b97edc5f4f2bc276d920b